Code/Tools/AssetProcessor/native/ui/SourceAssetDetailsPanel.h

```diff
@@ -34,12 +34,14 @@ namespace AssetProcessor
         explicit SourceAssetDetailsPanel(QWidget* parent = nullptr);
         ~SourceAssetDetailsPanel() override;
 
+        void SetIsIntermediateAsset() { m_isIntermediateAsset = true; }
+
     public Q_SLOTS:
         void AssetDataSelectionChanged(const QItemSelection& selected, const QItemSelection& deselected);
 
     protected:
         void ResetText();
-        void SetDetailsVisible(bool visible);
+        void SetDetailsVisible(bool visible, bool isIntermediateAsset);
```

Comment:
Am unclear why we have SetIsIntermediateAsset() and SetDetailsVisible() also taking the flag isIntermediateAsset()

Why do we not call:
* SetIsIntermediateAsset()
* SetDetailsVisible() with old data

Not clear why we are coupling things together in SetDetailsVisible.

Reply:
Fixed, removed it